Makefile.toml

```diff
@@ -109,6 +117,15 @@ dependencies = [
 	"action-build-debug",
 ]
 
+[tasks.build-examples]
+description = "## Build (release-mode) project example(s); usage: `cargo make (build-examples | examples | example) [EXAMPLE]...`"
+category = "[project]"
+dependencies = [
+	"core::pre-build",
```

Comment:
This either has a tab or an extra indent.

Reply:
I see that ... the white space formatting is all over the map within that file (and a few others in the repo).

I've found and fixed the formatting throughout the repo and included it as a whites space cleanup commit.

I also included a quick Windows BAT script which uses `pcregrep` to test for issues. If it's deemed useful, I'll write a corresponding bash/sh script equivalent and add something to the "Style" CICD checks for the future.

Reply:
... And, if so, I should also add an EditorConfig file to suggest to devs how the files should be formatted (and support that construction, at least within reasonably smart editors).

Reply:
... EditorConfig configuration file added